modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/PassThroughConstants.java

```diff
@@ -234,4 +234,12 @@
 
     //Check for invalid json message by parsing the input message
     public static final String FORCE_JSON_MESSAGE_VALIDATION = "force.json.message.validation";
+
+    /**constants for the EI observability*/
+    public static final String CORRELATION_ENABLE_STATE = "true";
```

Comment:
I think we can get rid of these two constants when property checks are refactored.